tests/autoclassifier/test_sum.py

```diff
@@ -0,0 +1,16 @@
+
+from gradsflow.autoclassifier import AutoSummarisation
+
+from unittest.mock import MagicMock
+
+
+def test_model():
+    datamodule = MagicMock()
+    model = AutoSummarisation(
+        datamodule,
+        max_epochs=1,
+        timeout=5,
+        suggested_backbones="sshleifer/distilbart-cnn-12-6",
+        n_trials=1,
+    )
+    model.hp_tune()
```

Comment:
`model.hp_tune()` will try to train the model with MagicMock class. We can remove it from here.

Reply:
Should we define datamodule as none then?

Reply:
Oh nevermind I got what you mean now!